paper/paper.md

```diff
@@ -22,7 +22,7 @@ bibliography: paper.bib
 
 # Summary
 
-There has been a remarkable effort in the last years in
+There has been a remarkable effort in recent years in
 the robotics community to formulate estimation problems
 properly. This is motivated by an increasing demand for
```

Comment:
Would it be helpful to a general reader to add either a small (1 sentence) description of the problems OR perhaps point at the mathematical paper for more details? As I'm not familiar with this area I don't really understand what the package is for.